### PR TITLE
unexport and poll functionality

### PR DESCRIPTION
**Pull request by posborne**
CC @nastevens 

Also, fix up a few minor things with export (was not
respecting export flag previously).

Signed-off-by: Paul Osborne <osbpau@gmail.com>

### CODE REVIEW

**Comment by posborne** [comment]
I decided to piggy-back the poll changes I had ready to go with this PR to allow for review all-at-once.

File: src/commands/gpio_exportall.rs [hunk 1]
```diff
@@ -11,7 +11,8 @@ pub fn main(config: &GpioConfig, opts: &GpioExportAllOptions) {
         None => config.get_symlink_root(),
     };
 
-    for pin in config.get_pins() {
+    // export all pins except those fork which export is set to false
```
Comment (martinsp) [inline src/commands/gpio_exportall.rs]: typo 'fork' -> 'for' ?

Comment (posborne) [inline src/commands/gpio_exportall.rs]: Good catch :)

**Comment by posborne** [comment]
CC @varzac.  Go ahead and review this.  If you are going to implement status, then we'll probably want this merged first (although they are probably not too directly related).

**Pull request merged**